src/v2/components/IconSquare.tsx

```diff
@@ -24,18 +32,26 @@ export type IconSquareProps = {
 
 export const IconSquare: React.FC<IconSquareProps> = (props) => {
   const { icon, onPress } = props;
+  const { theme } = useTheme();
 
   if (onPress) {
     return (
-      <StyledTouchableOpacity onPress={onPress} testID='icon-square'>
-        <Image source={icon} style={{ height: 15, width: 15 }} />
-      </StyledTouchableOpacity>
+      <TouchableOpacity
+        onPress={onPress}
+        style={[styles.container, theme.iconSquare.container]}
+        testID='icon-square'
+      >
+        <Image source={icon} style={styles.image} />
+      </TouchableOpacity>
     );
   }
 
   return (
-    <StyledView testID='icon-square'>
-      <Image source={icon} style={{ height: 15, width: 15 }} />
-    </StyledView>
+    <View
+      style={[styles.container, theme.iconSquare.container]}
```

Comment:
As discussed previously, should we add themeKey here as prop? `iconSquare.container`? @virdesai @nhannah

Reply:
so `theme[themeKey] || theme.iconSquare.container`? think that'd then require us to have the dot notation drilling function support on both the main style on Chat like it currently is and also each individual component

Reply:
We have added display name with bracket notation for theme path.